Tensile/KernelWriterAssembly.py

```diff
@@ -3763,6 +3763,42 @@ def openPersistentLoop(self, kernel):
     if kernel["PersistentKernel"] or kernel["StreamK"]:
       if kernel["StreamK"]:
         # Workload calculations
+        if kernel["StreamKXCCMapping"]:
+          sXCC = self.sgprPool.checkOut(1, "XCC", preventOverflow=0)
+          sGridC = self.sgprPool.checkOut(1, "sGridC", preventOverflow=0)
+          sGridF = self.sgprPool.checkOut(1, "sGridF", preventOverflow=0)
+          sGridM = self.sgprPool.checkOut(1, "sGridM", preventOverflow=0)
+          stmp = None
+          sqtmp = None
+          divisor = kernel["StreamKXCCMapping"]
+          if ((divisor & (divisor - 1)) != 0): # Need temp registers if not power of 2
+            stmp = self.sgprPool.checkOut(3, "stmp", preventOverflow=0)
+            sqtmp = stmp + 2
```

Comment:
This is overwrapping with tmpSgpr+ 2 in scalarStaticDivideAndRemainder.
Looks like using same sgpr for qReg and tmpSgpr+2 does not actually cause an issue, but it might be better to put some comment so that using same reg for qReg and tmpSgpr+2 is intentional.

Reply:
Updated